pkg/ibmvpc/connectivityAnalysis_test.go

```diff
@@ -1,443 +0,0 @@
-/*
```

Comment:
Why was this file removed?

Reply:
Adi agreed it is not necessary.

Reply:
Should we consider keeping the test, but changing the expected attributes to be compared with expected connectivity not as textual hard-coded output? 
can be in a separate issue/PR.

Reply:
I dont see the point, much more complex connectivity cases are tested, and the output is checked by text.
the tests of this files are basically redundant, and more high maintained.

Reply:
these are more local tests, that do not run the entire flow.. it may be easier for debug and for capturing what is tested in this context, as opposed to e2e tests with json inputs.

Reply:
I restored the file, and hard-codded changed the expected result.
Now we can later move the the expected result to files, and write code that update these file.